libcxx/test/std/strings/basic.string/string.ops/string_substr/subview.pass.cpp

```diff
@@ -0,0 +1,88 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// REQUIRES: std-at-least-c++26
+
+// <string>
+
+// constexpr basic_string_view<_CharT, _Traits> subview(size_type __pos = 0, size_type __n = npos) const;
```

Comment:
Missing test coverage:
- check that the method is marked as `const`
- check that we throw an exception when we're calling this with out-of-range arguments (there are various combinations of out-of-range arguments)